packages/button/src/__demo__/examples/iconOnly.js

```diff
@@ -29,25 +29,27 @@ export default {
   title: 'Icon-only Buttons',
   description:
     'Buttons that contain only Icons can use either `iconStart` or `iconEnd` props and must have an `aria-label` provided.',
-  scope: { Button, IconQueue, DemoLayout },
+  scope: { Button, IconCloud, DemoLayout },
   source: `() => {
-  const icon = <IconQueue color="currentColor" />;
+  const icon = <IconCloud />;
 
   return <DemoLayout>
     {/* Icon as prop; no text. aria-label applied to Button. */}
-    <Button iconStart={icon} aria-label="Add new" />
+    <Button iconStart={icon} aria-label="Cloud" />
     {/* Icon as child; no text. aria-label applied to Button. Note that the size of the icon must be provided in this case. */}
```

Comment:
This is surprisingly hard to fix. There are three relevant cases to consider:

1. Icon-only Button, via props
2. Icon-only Button, via children
3. Text-only Button

In all three, the Icon needs padding. In (2), the Icon is inside a Content, which should not have padding... but in (3) that same Content _does_ need padding. Without inspecting the children, there’s no way to style both (2) and (3).